reflex/components/recharts/polar.py

```diff
@@ -70,11 +71,11 @@ class Pie(Recharts):
     # Valid children components
     _valid_children: List[str] = ["Cell", "LabelList"]
 
-    # fill color
-    fill: Var[str]
+    # Stoke color
+    stroke: Var[Union[str, Color]] = Var.create_safe(Color("accent", 9))
 
-    # stroke color
-    stroke: Var[str]
+    # Fill color
+    fill: Var[str] = Var.create_safe(Color("accent", 3))
```

Comment:
```suggestion
    fill: Var[Union[str, Color]] = Var.create_safe(Color("accent", 3))
```

there's another one like this below too